generals.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, userid, username, mode="1v1", gameid=None,
 
         elif mode == "ffa":
             self._send(["play", userid, _BOT_KEY])
+            self._send(["set_force_start", gameid, force_start])
```

Comment:
Thanks for the fix! Instead of adding this line, please could you remove the "if gameid:" on line 69?